US975260: Refactor HPE references

raghavendran-gopalakrishnan opened the pull request:
https://internal.almoctane.com/ui/entity-navigation?p=131002/6001&entityType=work_item&id=975260

raghavendran-gopalakrishnan committed:
US975260: Refactor HPE references to opentext

https://internal.almoctane.com/ui/entity-navigation?p=131002/6001&entityType=work_item&id=975260

raghavendran-gopalakrishnan commented:
Yet to update the javadocs.

buildmachine-sou-jenkins2 commented:
A developer build has not yet been created for this branch. Click [here](https://sou-jenkins2.swinfra.net/job/Create-New-Build/parambuild/?GITHUB_SERVER=github.com&GITHUB_ORGANISATION=WorkerFramework&REPO_NAME=worker-framework&BRANCH_NAME=975260) to go ahead and create the build...

buildmachine-sou-jenkins2 commented:
CI Build Link:
https://sou-jenkins2.swinfra.net/job/WorkerFramework/job/WorkerFramework~worker-framework~975260~CI

buildmachine-sou-jenkins2 commented:
The Documentation QA site for this branch has been built:
https://pages.github.houston.softwaregrp.net/sepg-docs-qa/WorkerFramework_CI_worker-framework_975260

raghavendran-gopalakrishnan committed:
Revert "US975260: Refactor HPE references to opentext"

This reverts commit 12eefcd631516f1d6020bef78b9922ab567736e5.

raghavendran-gopalakrishnan committed:
Merge branch 'develop' into 975260

raghavendran-gopalakrishnan committed:
US975260: Update to latest caf-commons

https://internal.almoctane.com/ui/?p=131002/6001#/entity-navigation?entityType=work_item&id=975260

raghavgopal commented on pom.xml:
```diff
@@ -211,64 +211,64 @@
+                <groupId>com.github.cafapi.common.api</groupId>
                 <artifactId>caf-api</artifactId>
-                <version>3.2.0-SNAPSHOT</version>
+                <version>4.0.0-975260-SNAPSHOT</version>
```
```suggestion
                <version>4.0.0-SNAPSHOT</version>
```

raghavgopal commented on pom.xml:
```diff
@@ -211,64 +211,64 @@
+                <groupId>com.github.cafapi.common.ciphers</groupId>
                 <artifactId>cipher-null</artifactId>
-                <version>3.2.0-SNAPSHOT</version>
+                <version>4.0.0-975260-SNAPSHOT</version>
```
```suggestion
                <version>4.0.0-SNAPSHOT</version>
```

raghavgopal commented on pom.xml:
```diff
@@ -211,64 +211,64 @@
+                <groupId>com.github.cafapi.common.codecs</groupId>
                 <artifactId>codec-json</artifactId>
-                <version>3.2.0-SNAPSHOT</version>
+                <version>4.0.0-975260-SNAPSHOT</version>
```
```suggestion
                <version>4.0.0-SNAPSHOT</version>
```

raghavgopal commented on pom.xml:
```diff
@@ -211,64 +211,64 @@
-                <version>3.2.0-SNAPSHOT</version>
+                <groupId>com.github.cafapi.common.config</groupId>
+                <artifactId>config-source</artifactId>
+                <version>4.0.0-975260-SNAPSHOT</version>
```
```suggestion
                <version>4.0.0-SNAPSHOT</version>
```

raghavgopal commented on pom.xml:
```diff
@@ -211,64 +211,64 @@
+                <groupId>com.github.cafapi.common.config</groupId>
                 <artifactId>config-decoder</artifactId>
-                <version>3.2.0-SNAPSHOT</version>
+                <version>4.0.0-975260-SNAPSHOT</version>
```
```suggestion
                <version>4.0.0-SNAPSHOT</version>
```

raghavgopal commented on pom.xml:
```diff
@@ -211,64 +211,64 @@
+                <groupId>com.github.cafapi.common.config.sources</groupId>
                 <artifactId>config-file</artifactId>
-                <version>3.2.0-SNAPSHOT</version>
+                <version>4.0.0-975260-SNAPSHOT</version>
```
```suggestion
                <version>4.0.0-SNAPSHOT</version>
```

raghavgopal commented on pom.xml:
```diff
@@ -211,64 +211,64 @@
-                <version>3.2.0-SNAPSHOT</version>
+                <groupId>com.github.cafapi.common.bootstrapconfigs</groupId>
+                <artifactId>bootstrapconfig-system</artifactId>
+                <version>4.0.0-975260-SNAPSHOT</version>
```
```suggestion
                <version>4.0.0-SNAPSHOT</version>
```

raghavgopal commented on pom.xml:
```diff
@@ -211,64 +211,64 @@
+                <groupId>com.github.cafapi.common.decoders</groupId>
                 <artifactId>decoder-js</artifactId>
-                <version>3.2.0-SNAPSHOT</version>
+                <version>4.0.0-975260-SNAPSHOT</version>
```
```suggestion
                <version>4.0.0-SNAPSHOT</version>
```

raghavgopal commented on pom.xml:
```diff
@@ -211,64 +211,64 @@
+                <groupId>com.github.cafapi.common.util</groupId>
                 <artifactId>util-moduleloader</artifactId>
-                <version>3.2.0-SNAPSHOT</version>
+                <version>4.0.0-975260-SNAPSHOT</version>
```
```suggestion
                <version>4.0.0-SNAPSHOT</version>
```

raghavgopal commented on pom.xml:
```diff
@@ -211,64 +211,64 @@
+                <groupId>com.github.cafapi.common.util</groupId>
                 <artifactId>util-naming</artifactId>
-                <version>3.2.0-SNAPSHOT</version>
+                <version>4.0.0-975260-SNAPSHOT</version>
```
```suggestion
                <version>4.0.0-SNAPSHOT</version>
```

raghavgopal commented on pom.xml:
```diff
@@ -211,64 +211,64 @@
+                <groupId>com.github.cafapi.common.util</groupId>
                 <artifactId>util-ref</artifactId>
-                <version>3.2.0-SNAPSHOT</version>
+                <version>4.0.0-975260-SNAPSHOT</version>
```
```suggestion
                <version>4.0.0-SNAPSHOT</version>
```

raghavendran-gopalakrishnan committed:
Apply suggestions from code review

raghavendran-gopalakrishnan committed:
Remove HPE references

raghavendran-gopalakrishnan committed:
Merge branch '975260' of https://github.com/WorkerFramework/worker-framework into 975260

raghavendran-gopalakrishnan commented:
To do:
Consolidate javadocs of caf-commons and worker-framework.
Reversion to major

raghavendran-gopalakrishnan committed:
Update POM versions to 9.0.0-SNAPSHOT

raghavendran-gopalakrishnan committed:
Missed stuffs

raghavendran-gopalakrishnan committed:
Merge branch '975260' of https://github.com/WorkerFramework/worker-framework into 975260

raghavendran-gopalakrishnan committed:
Release notes updated

raghavendran-gopalakrishnan committed:
Merge branch 'develop' into 975260

# Conflicts:
#	pom.xml
#	release-notes-8.2.0.md

dermot-hardy commented on worker-core/pom.xml:
```diff
@@ -21,14 +21,14 @@
     <modelVersion>4.0.0</modelVersion>
 
-    <groupId>com.github.workerframework</groupId>
+    <groupId>com.github.workerframework.worker</groupId>
```
This extra `.worker` is bothersome.  Put these in a spreadsheet and let's go through them.

raghavgopal replied:
Ok let's review them today.

raghavendran-gopalakrishnan committed:
Dropped 'worker' from groupId, package names

raghavendran-gopalakrishnan committed:
Merge branch 'develop' into 975260

# Conflicts:
#	docs/pom.xml
#	pom.xml
#	standard-worker-container/pom.xml
#	util-rabbitmq/pom.xml
#	worker-api/pom.xml
#	worker-caf/pom.xml
#	worker-configs/pom.xml
#	worker-core/pom.xml
#	worker-default-configs/pom.xml
#	worker-queue-rabbit/pom.xml
#	worker-store-fs/pom.xml
#	worker-store-http/pom.xml
#	worker-store-mem/pom.xml
#	worker-store-s3/pom.xml
#	worker-test/pom.xml
#	worker-tracking-report/pom.xml

dermot-hardy commented on pom.xml:
```diff
@@ -211,115 +211,115 @@
             </dependency>
             <dependency>
-                <groupId>com.github.cafapi</groupId>
+                <groupId>com.github.cafapi.common.api</groupId>
```
This makes me realize we made a mistake in `caf-common`.  We need to discuss this.

raghavgopal replied:
Ok it was supposed to be just `com.github.cafapi.common` for all caf-common artifacts.

dermot-hardy replied:
Right

raghavgopal replied:
Will fix them.

dermot-hardy commented on worker-core/readme.md:
````diff
@@ -83,7 +83,7 @@ the current input queue. Default is True.
  
  ```
- java -cp "*" com.hpe.caf.worker.core.WorkerApplication server [settings.yaml]
+ java -cp "*" WorkerApplication server [settings.yaml]
````
```suggestion
 java -cp "*" com.github.workerframework.core.WorkerApplication server [settings.yaml]
```

dermot-hardy commented on worker-core/readme.md:
```diff
@@ -707,14 +707,14 @@ different results.
  import com.hpe.caf.api.worker.DataSource;
- import com.hpe.caf.api.worker.WorkerException;
- import com.hpe.caf.api.worker.WorkerFactoryProvider;
+ import WorkerException;
+ import WorkerFactoryProvider;
```
```suggestion
 import com.github.workerframework.api.WorkerException;
 import com.github.workerframework.api.WorkerFactoryProvider;
```

dermot-hardy commented on worker-core/readme.md:
```diff
@@ -755,14 +755,14 @@ different results.
- import com.hpe.caf.test.worker.shared.TestWorkerResult;
+ import com.github.cafapi.common.api.Codec;
+ import com.github.cafapi.common.api.CodecException;
+ import Worker;
+ import WorkerException;
```
```suggestion
 import com.github.workerframework.api.Worker;
 import com.github.workerframework.api.WorkerException;
```

dermot-hardy commented on worker-core/readme.md:
```diff
@@ -849,8 +849,8 @@ different results.
  case, the file will be called
- `META-INF/services/com.hpe.caf.api.worker.WorkerFactoryProvider` and will
- have the line `com.hpe.caf.test.worker.TestWorkerFactoryProvider`.
+ `META-INF/services/WorkerFactoryProvider` and will
```
```suggestion
 `META-INF/services/com.github.workerframework.api.WorkerFactoryProvider` and will
```

raghavendran-gopalakrishnan committed:
Apply suggestions from code review

Co-authored-by: Dermot Hardy <dhardy2@opentext.com>

raghavendran-gopalakrishnan committed:
Fixed caf-common groupIds

raghavendran-gopalakrishnan committed:
Updated version

raghavgopal commented on pom.xml:
```diff
@@ -211,115 +211,115 @@
+                <groupId>com.github.cafapi.common</groupId>
                 <artifactId>caf-api</artifactId>
-                <version>3.2.0-684</version>
+                <version>4.0.0-975260-1-SNAPSHOT</version>
```
```suggestion
                <version>4.0.0-SNAPSHOT</version>
```

raghavgopal commented on pom.xml:
```diff
@@ -211,115 +211,115 @@
+                <groupId>com.github.cafapi.common</groupId>
                 <artifactId>cipher-null</artifactId>
-                <version>3.2.0-684</version>
+                <version>4.0.0-975260-1-SNAPSHOT</version>
```
```suggestion
                <version>4.0.0-SNAPSHOT</version>
```

raghavgopal commented on pom.xml:
```diff
@@ -211,115 +211,115 @@
+                <groupId>com.github.cafapi.common</groupId>
                 <artifactId>codec-json</artifactId>
-                <version>3.2.0-684</version>
+                <version>4.0.0-975260-1-SNAPSHOT</version>
```
```suggestion
                <version>4.0.0-SNAPSHOT</version>
```

raghavgopal commented on pom.xml:
```diff
@@ -211,115 +211,115 @@
-                <version>3.2.0-684</version>
+                <groupId>com.github.cafapi.common</groupId>
+                <artifactId>config-source</artifactId>
+                <version>4.0.0-975260-1-SNAPSHOT</version>
```
```suggestion
                <version>4.0.0-SNAPSHOT</version>
```

raghavgopal commented on pom.xml:
```diff
@@ -211,115 +211,115 @@
+                <groupId>com.github.cafapi.common</groupId>
                 <artifactId>config-decoder</artifactId>
-                <version>3.2.0-684</version>
+                <version>4.0.0-975260-1-SNAPSHOT</version>
```
```suggestion
                <version>4.0.0-SNAPSHOT</version>
```

raghavgopal commented on pom.xml:
```diff
@@ -211,115 +211,115 @@
+                <groupId>com.github.cafapi.common</groupId>
                 <artifactId>config-file</artifactId>
-                <version>3.2.0-684</version>
+                <version>4.0.0-975260-1-SNAPSHOT</version>
```
```suggestion
                <version>4.0.0-SNAPSHOT</version>
```

raghavgopal commented on pom.xml:
```diff
@@ -211,115 +211,115 @@
-                <version>3.2.0-684</version>
+                <groupId>com.github.cafapi.common</groupId>
+                <artifactId>bootstrapconfig-system</artifactId>
+                <version>4.0.0-975260-1-SNAPSHOT</version>
```
```suggestion
                <version>4.0.0-SNAPSHOT</version>
```

raghavgopal commented on pom.xml:
```diff
@@ -211,115 +211,115 @@
+                <groupId>com.github.cafapi.common</groupId>
                 <artifactId>decoder-js</artifactId>
-                <version>3.2.0-684</version>
+                <version>4.0.0-975260-1-SNAPSHOT</version>
```
```suggestion
                <version>4.0.0-SNAPSHOT</version>
```

raghavgopal commented on pom.xml:
```diff
@@ -211,115 +211,115 @@
+                <groupId>com.github.cafapi.common</groupId>
                 <artifactId>util-moduleloader</artifactId>
-                <version>3.2.0-684</version>
+                <version>4.0.0-975260-1-SNAPSHOT</version>
```
```suggestion
                <version>4.0.0-SNAPSHOT</version>
```

raghavgopal commented on pom.xml:
```diff
@@ -211,115 +211,115 @@
+                <groupId>com.github.cafapi.common</groupId>
                 <artifactId>util-naming</artifactId>
-                <version>3.2.0-684</version>
+                <version>4.0.0-975260-1-SNAPSHOT</version>
```
```suggestion
                <version>4.0.0-SNAPSHOT</version>
```

raghavgopal commented on pom.xml:
```diff
@@ -211,115 +211,115 @@
+                <groupId>com.github.cafapi.common</groupId>
                 <artifactId>util-ref</artifactId>
-                <version>3.2.0-684</version>
+                <version>4.0.0-975260-1-SNAPSHOT</version>
```
```suggestion
                <version>4.0.0-SNAPSHOT</version>
```

raghavendran-gopalakrishnan committed:
consume caf-common latest
